bin/validate_case_api.js

```diff
@@ -0,0 +1,112 @@
+const process = require('process');
+const fs = require('fs');
+const walkSync = require('walk-sync');
+const request = require('request');
+
+const BASE_IA_CCD_DIR = process.env.IA_CCD_DIR !== undefined ? process.env.IA_CCD_DIR : '../ia-ccd-definitions/';
```

Comment:
I will add remote reading for CCD definitions as default and leave local directory reading as optional within next PR.